cmd/utils/utils.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/spf13/pflag"
 )
 
-const ProgramName = "go-blueprint"
+const ProgramName = "go-blueprint create"
```

Comment:
i think you remove create from here. this non-interactive command tip message issue resolved in #123

Reply:
tnx